### DISCUSSION TITLE
Hotmart - Server returned 403 Forbidden (access denied)

### DISCUSSION THREAD

**Discussion by WillMenezess** [aclap-dev/video-downloadhelper#549] [Help and Q&A]
### Your question

Hello Everyone, I bought the video downloadhelper license to be able to download the videos from my course purchased on hotmart. Some videos I can download, others give me this error:

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://hotmart.com/pt-BR/club/simplifica-treinamentos/products/904229/content/E4zKNZG9el",
  "error": {
    "details": "SideDownload error: : [in#1 @ 000001f2e241a740] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://vod-akm.play.hotmart.com/video/X7LyErd9LA/hls/X7LyErd9LA-1681292602000-audio_2=126660.m3u8?hdntl=exp=1716493112~acl=/*~data=hdntl~hmac=c941afe809e8d0bd5347dea86a85f715216873c6b2259538eba17558cf742717&app=f57b1bb2-30e3-4fd0-9c59-cfe00909f280&tokenParams=eyJhbGciOiJub25lIn0.eyJ0cmFuc2NvZGVkT2JqZWN0S2V5cyI6WyJ2aWRlby9YN0x5RXJkOUxBL21wNC83OWI3M2JkZS0yMGNjLTQzY2ItYmQwMS1hMGFkYWZmNjI2ZTcubXA0IiwidmlkZW8vWDdMeUVyZDlMQS9tcDQvZWIyMGM0ZmMtOTZlYy00NDRjLWIzZWMtNzJjMjkxYzg0MzUxLm1wNCIsInZpZGVvL1g3THlFcmQ5TEEvbXA0LzhlYWFjMTM3LTdjZGQtNDUwZS1iOGE5LTMxMzg5NDVlMDJkMi5tcDQiLCJ2aWRlby9YN0x5RXJkOUxBL21wNC8zNDBkM2EzMy0zMGZmLTRiYjctOTAwNC1jN2Q3ZWMzZjk2NDkubXA0IiwidmlkZW8vWDdMeUVyZDlMQS9tcDQvNTBmZTU3ZWQtMWM1Zi00ODRiLWI2ZDAtYzhhYjdhNWMxODMyLm1wNCJdLCJ1cGRhdGVEYXRlIjoxNjgxMjkyNjAyMDAwLCJjZG5VcmwiOiJodHRwczovL3ZvZC1ha20ucGxheS5ob3RtYXJ0LmNvbSIsImRybSI6ZmFsc2UsInN1YiI6Ilg3THlFcmQ5TEEiLCJpYXQiOjE3MTI2NzAyNTZ9..\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "downloadable_id": "downloadable_5906585968896656",
    "error": "coapp_failure",
    "id": "coapp_failure_518e9ad1-6a59-478f-83f4-371967ed65e2"
  }
}

![Captura de tela 2024-05-22 164038](https://github.com/aclap-dev/video-downloadhelper/assets/115901716/9fc71c7e-9777-4d4d-954c-6c911dabc73d)

Can someone help me?

### Addon & CoApp version and details

version: 9.0.1.13
target: google
channel: stable
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"bfe84202f00c4392"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "download_directory": "C:\\Users\\Willm\\Downloads",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 10
}

**Comment by petaqui** [comment]
Having same issue with this platform. Previously it worked
